livetree.c

```diff
@@ -3,6 +3,8 @@
  * (C) Copyright David Gibson <dwg@au1.ibm.com>, IBM Corporation.  2005.
  */
 
+#include <libfdt/libfdt.h>
+
```

Comment:
Oof.  We don't currently use libfdt within dtc, which is sort-of deliberate (they provide a cross-check to each other).  There's a case to be made to drop that policy and instead use libfdt for much of the flat tree logic within dtc.  However, I'd really prefer to do that as an overall change, using it for as much as we can, rather than adding small pieces of libfdt usage ad-hoc.

..also, I can't actually see what you're using this for.

Reply:
Maybe this is only a development relict, will try without.